### ISSUE TITLE
White Text/White Background on Unloaded Tabs

### ISSUE THREAD

**Issue by McCovican** [Quicksaver/Tab-Groups#503]
First up, I'm not sure if this is a duplicate of #250 but this is on Win7 FF50 with a light theme (Simple White X if it's relevant). In any case, this is still an issue for me. Unread tabs have white/grey text, on the same coloured background. You can see in the screenshot it's unreadable entirely on one tab, and even when the tab is hovered over, it's close enough to unreadable. It's not an issue once the tab has been loaded.

![tabgroups](https://cloud.githubusercontent.com/assets/11760565/20728646/997e2044-b676-11e6-9e2e-53d750690cfa.jpg)

**Comment by Quicksaver** [comment]
Do you have Tab Mix Plus? If so, have you set it to color the text of unloaded tabs white?

**Comment by McCovican** [comment]
Hi,

Sorry for the delay. Initially I thought this wasn't the case - I actually use Classic Theme Restorer to alter the colour of my tabs, and the colours I used with that didn't remotely match what was being displayed in Tab Groups. However, I took a look at Tab Mix Plus on the off-chance, and you were bang-on. It had #ffffff set for unloaded tabs. Seems CTR overrides Tab Mix's settings when displaying the actual tabs, but still alters it somewhere else that's relevant to Tab Groups. Disabled all the colour settings in Tab Mix, and all is now working properly.

Cheers!
